istioctl/cmd/istioctl_test.go

```diff
@@ -15,34 +15,18 @@
 package cmd
 
 import (
-	"bytes"
-	"regexp"
-	"strings"
 	"testing"
 
-	"github.com/spf13/cobra"
-
-	"istio.io/istio/pilot/test/util"
+	util2 "istio.io/istio/istioctl/pkg/util"
```

Comment:
can we cleanup the util2

Reply:
Thanks, all the `util2` usages have been refactored to `istioctlutil`